app/src/shared/interfaces.ts

```diff
@@ -0,0 +1,8 @@
+export type SpecOperation = 'merge' | 'filter';
```

Comment:
I think we should add overlay here, and perhaps there should be different interfaces for each operation type? for example, if we implement overlay, it would not need a defaults file or a conflictresolutions file, just a spec and an overlay. (Maybe you already meant to do this from the recent changes to the OpenAPI contract in this pr: https://github.com/googleinterns/spec-math/pull/5)

Reply:
Will be adding for now